devices/b-parasite/b-parasite.json

```diff
@@ -0,0 +1,415 @@
+{
+  "schema": "devcap1.schema.json",
+  "uuid": "7ad2d274-5bb0-40f1-926b-4d9d059ce393",
+  "manufacturername": "b-parasite",
+  "modelid": "b-parasite",
+  "product": "b-parasite",
```

Comment:
Maybe you can spend just a few words on what this device is? :)

Reply:
Done in cf6326537796a484940a8690484c3257db3be914